src/audio/dai.c

```diff
@@ -838,16 +851,17 @@ static void dai_cache(struct comp_dev *dev, int cmd)
 static struct comp_driver comp_dai = {
 	.type	= SOF_COMP_DAI,
 	.ops	= {
-		.new		= dai_new,
-		.free		= dai_free,
-		.params		= dai_params,
-		.trigger	= dai_comp_trigger,
-		.copy		= dai_copy,
-		.prepare	= dai_prepare,
-		.reset		= dai_reset,
-		.dai_config	= dai_config,
-		.position	= dai_position,
-		.cache		= dai_cache,
+		.new			= dai_new,
```

Comment:
Whitespace change should be separate commit. I want to only see the addition of .dai_get_hw_params in this commit.

Reply:
I have split it into two separate commits.